analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CryptographicKeyShouldNotBeTooShort.CSharp9.cs

```diff
@@ -72,6 +72,6 @@ public void GenerateKey()
         ec2.GenerateKey(ECCurve.NamedCurves.brainpoolP160t1); // Noncompliant {{Use a key length of at least 224 bits for EC cipher algorithm.}}
 
         ECDsaOpenSsl ec3 = new();
-        ec3.GenerateKey(ECCurve.NamedCurves.brainpoolP192t1); // FN, See: https://github.com/SonarSource/sonar-dotnet/issues/6164
+        ec3.GenerateKey(ECCurve.NamedCurves.brainpoolP192t1); // Noncompliant {{Use a key length of at least 224 bits for EC cipher algorithm.}}
```

Comment:
ECDsaOpenSsl.BaseType.ConstructedFrom 
_Before .Net 7_
Crypthography.ECDsaOpenSsl (which baseType is **Crypthography.ECDsa**)
_From .Net 7_
**Crypthography.ECAlgorythm**